src/components/user-dropdown-disclosure/types.ts

```diff
@@ -20,6 +20,7 @@ type UserDropdownDisclosureItem =
 	  }
 
 interface UserDropdownDisclosureProps {
+	className?: DropdownDisclosureProps['className']
```

Comment:
Total style nit: I think it's neat to have the optional properties at the bottom of interface definitions. Take it or leave it 😸

Reply:
Ah, I did't know that was your preference! I tend to alphabetize properties in most cases because it can make them easier to find when there are a lot of them. Might be a style thing to chat about as a group and explore automating. 😎